app/Http/Controllers/V2/Terrafund/TerrafundCreateGeometryController.php

```diff
@@ -0,0 +1,757 @@
+<?php
+
+namespace App\Http\Controllers\V2\Terrafund;
+
+use App\Http\Controllers\Controller;
+use App\Models\V2\PolygonGeometry;
+use App\Models\V2\Sites\CriteriaSite;
+use App\Models\V2\Sites\SitePolygon;
+use App\Models\V2\WorldCountryGeneralized;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Str;
+use Symfony\Component\Process\Process;
+
+class TerrafundCreateGeometryController extends Controller
+{
+    public function processGeometry(string $uuid)
+    {
+        $geometry = PolygonGeometry::where('uuid', $uuid)
+          ->select(DB::raw('ST_AsGeoJSON(geom) AS geojson'))
+          ->first();
+
+        $geojson = $geometry->geojson;
+
+        if ($geojson) {
+            return response()->json(['geometry' => $geojson], 200);
+        } else {
+            return response()->json(['error' => 'Geometry not found'], 404);
+        }
+    }
+
+    public function storeGeometry(Request $request)
+    {
+        $validatedData = $request->validate([
+          'geometry' => 'required|json',
+        ]);
+        $geometry = json_decode($request->input('geometry'));
+        $geom = DB::raw("ST_GeomFromGeoJSON('" . json_encode($geometry) . "')");
+        $uuid = Str::uuid();
+
+        DB::table('polygon_geometry')->insert([
+          'uuid' => $uuid,
+          'geom' => $geom,
+          'created_at' => now(),
+          'updated_at' => now(),
+        ]);
+
+        return response()->json(['uuid' => $uuid], 200);
```

Comment:
With the change above, this will change to 
```
return response()->json(['uuid' => $model->uuid], 200);
```

Reply:
👍🏼